network-store-model/src/main/java/com/powsybl/network/store/model/TwoWindingsTransformerAttributes.java

```diff
@@ -136,4 +136,42 @@ public class TwoWindingsTransformerAttributes extends AbstractAttributes impleme
 
     @Schema(description = "CGMES tap changer attributes list")
     private List<CgmesTapChangerAttributes> cgmesTapChangerAttributesList;
+
+    // @Override
+    // @JsonIgnore
+    // public List<Integer> getSideList() {
+    //     return List.of(0);
+    // }
+
+    @Override
+    public RatioTapChangerAttributes getRatioTapChangerAttributes(int side) {
+        if (side != 0) {
```

Comment:
This seems fishy, especially since the two sides of a twowindingstransformer are 1 or 2.

Reply:
removed